api/graphite.go

```diff
@@ -675,7 +676,7 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan expr.Plan)
 			for i, e := range exprs {
 				exprs[i] = strings.Trim(e, " '\"")
 			}
-			series, err = s.clusterFindByTag(ctx, orgId, exprs, int64(r.From))
+			series, err = s.clusterFindByTag(ctx, orgId, exprs, int64(r.From), maxSeriesPerReq-len(reqs))
```

Comment:
where is `reqs` updated?

Reply:
https://github.com/bloomberg/metrictank/pull/36/files/b6b419db4d99c11486ad0e9263cc13570b659504#diff-841baed3f3a566ff48a5a4a427defb30R708